packages/cli/src/commands/hydrogen/shortcut.ts

```diff
@@ -0,0 +1,73 @@
+import Command from '@shopify/cli-kit/node/base-command';
+import {renderFatalError, renderSuccess} from '@shopify/cli-kit/node/ui';
+import {execSync} from 'child_process';
+import Flags from '@oclif/core/lib/flags.js';
+import {fileURLToPath} from 'url';
+import path from 'path';
+
+export default class GenerateRoute extends Command {
```

Comment:
> Adding a global alias pointing to a locally-scoped dependencies feels very hacky and it can break easily

I think I've seen this in other libs (like Prisma, I think?) and I thought it was be best approach because you don't need to take care of global dependencies versioning and updating them, or making a new mini-CLI and maintaining it 🤔 
Like, if we end up adding the h2 shortcut, I think it should not create an extra maintenance burden for us. Otherwise I'm not sure if it's worth it.

I'm now thinking we should just ask devs to add `alias h2='$(npm bin)/shopify hydrogen'` to their bashrc/zshrc or whatever (or do it automatically for them). No idea for windows but perhaps there's a simple way.